cli.js

```diff
@@ -27,6 +28,25 @@ If file/glob is omitted, './package.json' file will be processed.
   )
 }
 
+function parseCliArguments() {
+  try {
+    return parseArgs({
```

Comment:
Just stylish, we can

````js
const {
	positionals: patterns,
	values: flags,
} = parseArgs({})

return {patterns, flags}
```

easier to use/understand.

Reply:
Example I wrote before [1](https://github.com/prettier/prettier/blob/97c8e3c18ff3299c76e0d4c1a7374ebce0735f00/scripts/release/parse-arguments.js#L4-L12), [2](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/fcd893465f8e3cc59d34d6d1db533cea94247a0c/test/integration/test.mjs#L33-L37)